OpenDreamRuntime.Tests/UnitTest1.cs

```diff
@@ -145,5 +145,24 @@ public class Tests {
             Assert.AreEqual(imageDefinition, obj.ObjectDefinition);
             Assert.Zero(runtime.ExceptionCount);
         }
+
+        [Test, Timeout(1000)]
```

Comment:
I ran this test a few times, it tends to take about 900ms and I even had it fail once. Should the timeout be increased?

Reply:
Yes, I'll web edit it to a generous 10 seconds as it'll ideally not fail most of the time